modules/nlu/src/views/api.ts

```diff
@@ -21,6 +23,10 @@ export const makeApi = (bp: { axios: AxiosInstance }): NLUApi => ({
   },
   fetchIntent: (intentName: string) => bp.axios.get(`/mod/nlu/intents/${intentName}`).then(res => res.data),
   createIntent: (intent: Partial<NLU.IntentDefinition>) => bp.axios.post(`/mod/nlu/intents`, intent),
+  renameIntent: (targetIntent: string, name: string) =>
```

Comment:
since POST to /intents/:targetIntent is pretty much a update intent, can you rename this func to updateIntent and that accepts `Partial<NLU.IntentDefinition>` as payload